Makefile

```diff
@@ -0,0 +1,7 @@
+lint:
+	@flake8
+
+test: lint
+	@python3 -m unittest
+
+.PHONY: test
```

Comment:
isn't 'lint' also PHONY?

Reply:
Thanks, have added that, and made test the default target. I agree having this be authoritative would be ideal. I'm not sure about whether "test" or "check" is more common practice for a standard Makefile target - I've seen both but would be happy to go with either.